Caboodle/Browser/Browser.android.cs

```diff
@@ -0,0 +1,82 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.Caboodle
+{
+    public partial class Browser
+    {
+        public static async Task OpenAsync(string uri)
+        {
+            if (string.IsNullOrEmpty(uri))
+            {
+                throw new ArgumentNullException($"Uri cannot be null or empty.");
+            }
+
+            await Browser.OpenAsync(new System.Uri(uri), BrowserLaunchingType.UriLauncher);
+
+            return;
+        }
+
+        public static async Task OpenAsync(string uri, BrowserLaunchingType launch_type)
+        {
+            if (string.IsNullOrEmpty(uri))
+            {
+                throw new ArgumentNullException($"Uri cannot be null or empty.");
+            }
+
+            await Browser.OpenAsync(new System.Uri(uri));
```

Comment:
Instead of `await` here, since you're just returning the result, you should remove `async` from the method and just `return OpenAsync (new System.Uri(uri));`